src/materialui/inputs.tsx

```diff
@@ -10,19 +10,105 @@ import {
 	Theme,
 	createStyles,
 	Modal,
+	List,
+	ListItem,
+	ListItemText,
+	Divider,
 } from '@material-ui/core';
 import injectSheet, { StyleSheet } from 'react-jss';
 import { DeleteIcon } from '../icons/delete';
 import classNames from 'classnames';
 import { HardDriveIcon } from '../icons/hard-drive';
 import { FileDefaultIcon } from '../icons/file-default';
-import { primary, baseDark, base, error } from '../colors';
+import { primary, baseDark, base, error, white } from '../colors';
 import { ModalWrap } from './modalWithBackButton';
 import { ModalBody2 } from './modalElements';
 import { FileImageIcon } from '../icons/file-image';
 import { FilePdfIcon } from '../icons/file-pdf';
 import { FileAudioIcon } from '../icons/file-audio';
 
+const multilineSelectStyles = (theme: Theme) =>
+	createStyles({
+		root: {
+			backgroundColor: baseDark,
+			borderRadius: '4px',
+			border: '1px solid #384656',
+			boxSizing: 'border-box',
+			maxWidth: '360px',
+			color: white,
+			fontSize: '14px',
+			overflow: 'auto',
+			lineHeight: '21px',
+			'&$focused': {
+				'&$focused:not($error):not($disabled)': {
+					border: `1px solid ${primary}`,
+					boxShadow: `0 0 3px 1px ${primary}`,
+				},
+			},
+		},
+		item: {
+			borderRadius: '4px',
+			fontSize: '14px',
+			lineHeight: '21px',
+			cursor: 'pointer',
+		},
+		itemAdd: {
+			background: 'none',
+			color: primary,
+			cursor: 'pointer',
+			lineHeight: '21px',
+			padding: '10px 16px',
+			marginBottom: 0
+		},
+	});
+
+export type MultilineSelectType = any;
+
+export const MultilineSelect = withStyles(multilineSelectStyles)((props: MultilineSelectType) => {
+	const { classes, multiselect, selected = [], items = [], onSelectUpdated, onAdd } = props;
+	const isSelected = (item: any) => selected.includes(item.key);
+	const handleItemClick = (item: any) => {
+		return (evt: React.SyntheticEvent) => {
+			if (evt) {
+				evt.preventDefault();
+			}
+			if (!onSelectUpdated) return;
+			if (!multiselect) {
+				if (isSelected(item)) {
+					return onSelectUpdated([]);
+				}
+				return onSelectUpdated([item.key]);
+			}
+			if (isSelected(item)) {
+				return onSelectUpdated(selected.filter((x: any) => x != item.key));
+			}
+			return onSelectUpdated([...selected, item.key]);
+		};
+	};
+	return (
+		<div>
+			<List className={props.classes.root}>
+				{items.map((item: any) => (
+					<ListItem
+						key={item.key}
+						className={classes.item}
+						selected={isSelected(item)}
+						onClick={handleItemClick(item)}
+					>
+						<ListItemText>{item.value}</ListItemText>
```

Comment:
Would be nice to reduce font size of item text (something like this).
`itemText: {
			'& span': {
				fontSize: '14px',
				lineHeight: '17px',
				padding: '5px 0'
			}
		}`